executor/gen_ppc64le.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+//go:build ppc64le && !freebsd && !darwin && !openbsd && !netbsd
```

Comment:
I think if you name the file `gen_linux_ppc64le.go` these constraints won't be needed. Go command automatically recognizes GOOS/GOARCH suffixes on source files.
Not sure why we didn't do it for amd64, probably it grew organically :)
Please also rename the x86 gen file.